packages/tasks/test/bridge/HopBridger.test.ts

```
@@ -370,7 +307,8 @@ describe('HopBridger', () => {
                     const tx = await task.call(token.address, requestedAmount, slippage, fee)
 
                     const deadline = (await currentTimestamp()).add(MAX_UINT256.div(10))
-                    const minAmountOut = amount.mul(fp(1).sub(slippage)).div(fp(1))
+                    const amountAfterFees = amount.sub(fee)
+                    const minAmountOut = amountAfterFees.mul(fp(1).sub(slippage)).div(fp(1))
```

Where is the max fee being set?

---

Nowhere. The config is being ignored
We discussed offline that it should raise an error when the max fee is not set and the fee is not 0